src/blocks/Item/schema.js

```diff
@@ -72,10 +74,15 @@ export default ({ data }) => {
       },
       theme: {
         title: 'Item theme',
-        choices: [
-          ['primary', 'Primary'],
-          ['secondary', 'Secondary'],
-          ['tertiary', 'Tertiary'],
+        widget: 'theme_picker',
+        colors: [
+          ...(config.settings && config.settings.themeColors
+            ? config.settings.themeColors.map(({ value, title }) => ({
+                name: value,
+                label: title,
+              }))
+            : []),
```

Comment:
@andreiggr We should have at least `primary` and `secondary` as fallback options in case volto-eea-website-theme is not present

Reply:
Hi @avoinea . The theme picker now has default themes: `Default, Primary, Secondary, Tertiary `